app/api-v1/routes/recipe.js

```diff
@@ -5,7 +5,27 @@ module.exports = function (recipeService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      if (!req.body) {
+        return res.status(400).json({ message: 'Invalid request' })
+      }
+
+      const { id, externalId, name, imageAttachmentId, material, alloy, price, requiredCerts, supplier } = req.body
+
+      const recipe = await recipeService.createRecipe({
+        id,
+        externalId,
+        name,
+        imageAttachmentId,
+        material,
+        alloy,
+        price,
+        requiredCerts,
+        supplier,
+      })
+      if (recipe == null) {
+        return res.status(200).json({ message: 'An error occurred' })
```

Comment:
why status when recipe = null? should it not say 404 - not found?

Reply:
I was referring to the swagger spec, happy to evaluate what is returned here.

![Screenshot 2022-04-29 at 09 15 48](https://user-images.githubusercontent.com/5233036/165908259-d89fe2fd-c350-4432-b7c3-34de2dd38d25.png)

Reply:
I think the bigger question is how can `recipe` be null here. It certainly shouldn't be a `404` as that only applies when a path resource doesn't exist